app/uk/gov/hmrc/incometaxsessiondata/domain/models/Session.scala

```diff
@@ -16,16 +16,43 @@
 
 package uk.gov.hmrc.incometaxsessiondata.domain.models
 
-import play.api.libs.json.{Format, Json}
+import play.api.libs.functional.syntax.{toFunctionalBuilderOps, unlift}
+import play.api.libs.json.{OFormat, __}
+import uk.gov.hmrc.mongo.play.json.formats.MongoJavatimeFormats
+
+import java.time.Instant
 
 case class Session(sessionID: String,
-                   mtditid: String,
-                   nino: String,
-                   saUtr: Option[String],
-                   clientFirstName: Option[String],
-                   clientLastName: Option[String],
-                   userType: String)
+                   sessionData: Option[SessionData] = None,
+                   lastUpdated: Instant = Instant.now)
 
+case class SessionData(mtditid: Option[String] = None,
```

Comment:
I would think that most of these fields are mandatory, not sure about first name and last name;